README.md

```diff
@@ -44,22 +43,22 @@ development teams - a skill which is currently in a very high demand.
         1.  Complete these challenges to see common vulnerabilities in action:
             1.  [OWASP WebGoat](https://github.com/WebGoat/WebGoat)
 
-            WebGoat has great tutorials, and will guide you through everything.
-             Its development version also enables you to learn how to fix common
+            WebGoat has great tutorials and will guide you through everything.
+             Its development version also teaches you how to fix common
              vulnerabilities in Java Web Applications. Additional lessons are
              available at [GitHub](https://github.com/WebGoat/WebGoat-Lessons).
 
             2.  [OWASP JuiceShop](https://github.com/bkimminich/juice-shop)
 
-            JuiceShop is a very good example of modern 'single-page'
+            JuiceShop is a good example of modern 'single-page'
             application, and it's challenges are fun!
 
         2.  Learn to use this tool, it's very useful:
             1.  [OWASP ZAP (Zed Attack Proxy)](https://github.com/zaproxy/zaproxy)
                 (or [Burp Suite](https://portswigger.net/burp) - commercial
                 analogue, extremely popular, and adequately priced)
 
-            These tools are very useful for looking into how web applications
+            ZAP contains tools for looking into how web applications
             work and for finding and reproducing potential vulnerabilities.
```

Comment:
Thank you so much for the awesome contribution!  

In this piece of text, I used plural to refer both ZAP and its alternatives (the only one mentioned is Burp at the moment). The idea was to give a choice for the reader, between ZAP and Burp. From this perspective, 'ZAP contains tools' is not what I meant - since 'tools' was referring to ZAP and Burp. Would it be ok with you to update the change accordingly?

Reply:
another FYI: looks like you are basing the change on the old branch, and not the 'master' (not sure how forks are updated in GitHub). The change looks good anyways, but this is a great opportunity to try 'git rebase' - will be happy to assist with that